src/inmanta/server/bootloader.py

```diff
@@ -106,6 +113,11 @@ async def start(self) -> None:
         await self.restserver.start()
         self.started = True
 
+    def start_loggers_for_extensions(self) -> FullLoggingConfig:
+        ctx = self.load_slices()
+        log_config_extenders = ctx.get_default_log_config_extenders()
```

Comment:
If I understand correctly, these will be retrieved in the order the extensions were registered in. Does this order ever matter ? How is it determined in the first place in case of multiple extensions?

Reply:
the order doesn't/shouldn't matter